client/my-sites/checkout/checkout-thank-you/redesign-v2/masterbar-styled/index.tsx

```diff
@@ -68,7 +68,7 @@ const MasterbarStyled = ( {
 			` }
 		/>
 		<WordPressLogoStyled size={ 24 } />
-		{ canGoBack && (
+		{ canGoBack && backText && onClick && (
```

Comment:
Ultimately, I don't think we need all these props. I would get rid of `canGoBack` so that we only rely on `backText` and `onClick` to determine whether to render a back button. However, given that this is currently being used by a number of pages, I'm going to keep `canGoBack` in place, we can clean things up after we're done refactoring!

Reply:
Yeah, I completely agree with you. It would be awesome to get rid of `canGoBack` since it's entirely redundant.